core/jwt_authentication.py

```diff
@@ -5,7 +5,9 @@
 from graphql_jwt.utils import get_credentials
 from graphql_jwt.exceptions import JSONWebTokenError
 from graphql_jwt.shortcuts import get_user_by_token
+from .apps import CoreConfig
 
+from datetime import datetime
```

Comment:
Just to be sure - should we use pydatetime instead of custom datetime (I'm thinking about one imported from core)?

Reply:
I've checked and it seems that it is only used in libraries code. Do we use it explicitly in the openIMIS?